### DISCUSSION TITLE
Mirth Connect Service stopping when unable to connect to SQL Server

### DISCUSSION THREAD

**Discussion by DavidFCSCR** [nextgenhealthcare/connect#5850] [Frequently Asked Questions]
Issue arisen where the Mirth Connect Service has stopped (I had to re-start it manually via Services), after the following error was recorded in the MirthLog:
ERROR 2023-07-09 21:04:11,982 [Main Server Thread] com.mirth.connect.server.Mirth: Error establishing connection to database, aborting startup. Network error IOException: Connection refused: connect
INFO  2023-07-09 21:04:12,076 [Shutdown Hook Thread] com.mirth.connect.server.Mirth: shutting down mirth due to normal request
 
Does anyone know if this is the expected outcome when the Mirth is unable to connect to a SQL Server, when the Mirth Channel is processing the message & attempts a connection to a sql server that is offline? Thanks

**Comment by kirbykn2** [comment]
Mirth cannot function without a connection to the mc db.  If you are connecting to another db from within your channel, this would not cause the Mirth service to stop.  Is it the case that you lost the connection to the backend mc db?

**Reply by pacmano1** [reply]
Question also asked here: https://forums.mirthproject.io/forum/mirth-connect/support/183614-why-did-mirth-connect-service-stop-when-attempting-to-connect-to-sql-server#post183615.    I suspect the same mirth.properties entry for retries likely applies to the engine as a whole, not just on start.
